ts/components/game/Player.tsx

```diff
@@ -44,6 +44,9 @@ class Player extends React.Component<PlayerProps, {}> {
 
 		/* Equipment */
 		var heroEntity = playEntities.filter(filterByCardType(CardType.HERO)).first();
+		if (!heroEntity) {
+			heroEntity = (this.props.entities.get(Zone.GRAVEYARD) || Immutable.Map<number, Entity>()).first();
```

Comment:
Oh I actually made a mistake here, this never filters for the hero. Lemme fix that...